tinker/templates/events/form.html

```diff
@@ -339,19 +339,29 @@ <h3 class="subtitle">{{ field.label }}</h3>
             };
 
 
-            function offCampusLocation() {
+            function offCampusLocation(clear_values) {
+                if (clear_values){ //Clears other options if the user switches in between them
+                    $("#off_campus_location").val('');
+                    $("#on_campus_location").val('none');
```

Comment:
why are you setting this as 'none'? just curious

Reply:
'none' is the value for the "-select-" option on the dropdown menu.